package.json

```diff
@@ -24,19 +24,19 @@
     "test": "yarn mocha handlers.test.js"
   },
   "dependencies": {
+    "@airswap/order-utils": "^0.3.6",
```

Comment:
does this mean that 0.3.7 will be picked up? That has the new ERC20_INTERFACE_ID in it.

Reply:
Yes `0.3.7` will be picked up. Will bump this here in a future publish.